atomic_accessor/atomic_accessor.md

```diff
@@ -612,12 +654,11 @@ constexpr basic-atomic-accessor(basic-atomic-accessor<OtherElementType, Referenc
 ```c++
 constexpr reference access(data_handle_type p, size_t i) const noexcept;
 ```
-[2]{.pnum} *Effects:* Equivalent to `return reference(p + i);`
-
+[2]{.pnum} *Effects:* Equivalent to `return p[i];`
 ```c++
 constexpr data_handle_type offset(data_handle_type p, size_t i) const noexcept;
 ```
-[3]{.pnum} *Effects:* Equivalent to `return p[i];`
+[3]{.pnum} *Effects:* Equivalent to `return reference(p + i);`
```

Comment:
Ugggg not so sure about the "`reference(...)`" part

Reply:
Could you elaborate?  That is the only place the passed in reference type is used.

Reply:
Fixed (I think).

Reply:
`offsets` returns `data_handle_type` (aka `ElementType*`) which is not constructible from `reference` (aka `atomic_ref<ElementType>` or alike)

Reply:
(would have sworn you said "can you elaborate")
anyway looks good now